gnucash/python/init.py

```diff
@@ -6,6 +6,8 @@
 require_version('Gtk', '3.0')
 from gi.repository import Gtk
 import os
+import gettext
+gettext.install("gnucash-pycons")
```

Comment:
I just put in a random module name here. Is there a naming convention ?

Reply:
It's gnucash. Changed that.

Reply:
@fellen thanks for the hint about GETTEXT_PACKAGE. I suppose that that is not accessible through SWIG at the moment. I came upon the question of how much of the gnucash core data should be made accessible to python. I found the existing bindings sw_core_utils are a middleground to make selected functions accessible. I imagine the initialization of python gettext could also take place in a swig file where it is easier to access a range of internal stuff but keep it out of the python user sphere to provide a cleaner API.

Reply:
I suppose it is also necessary to respect the possible locale changes in the gnucash environment files. I need to have a closer look at gnucash-bin.c. Maybe I can gnucash do the init/pref/environment stuff and copy the results like gettext.textdomain() etc. Or we just do the same things in python as we do in c.

In addition things work a bit differently in the python bindings than in the python shell module (I couldn't access environment variables in the latter even though gnc_environment_setup() gets called in gnucash_core.i and should have set them).